### ISSUE TITLE
Cannot reproduce results

### ISSUE THREAD

**Issue by vanoudh** [kexinhuang12345/clinicalBERT#9]
Hi Kexin,

I cannot reproduce the readmission results with pre-training. I only get 69% AUC.

I sampled 100000 notes for pre-training and used 75000 steps, following your paper.

I had to use pytorch-transformers v0.2.0 to be compatible with your code.

Any idea of the source of the discrepancy?

I can send you the full procedure if needed.

Thanks a lot
Marc

**Comment by kexinhuang12345** [comment]
The result is correct if using the pretraining script provided. The updated [paper](https://kexinhuang.com/s/main.pdf) result would be put in arxiv, it was accepted by the CHIL 2020 Workshop. 

Basically, it is due to a correlation between pretraining and finetuning when both are applied on the same dataset. In the old version of our paper, we suspect the performance gain is due to the accidental inclusion of the testing notes during pretraining sampling. The updated paper corrects the result.

Pretraining for Clinical Corpus is still useful, as shown by the [Clinical XLNet](https://arxiv.org/abs/1912.11975) paper, which follows the correct splitting method.

Closing for now!

**Comment by ynurmahomed** [comment]
Hi @vanoudh what loss value did you manage to achieve with pre-training?
